home/home.css

```diff
@@ -0,0 +1,150 @@
+/* ############################################# Slider-Css ################################# */
+
+.slider {
+  width: 100%;
+  height: 25vw;
+  border-radius: 10px;
+  overflow: hidden;
+  position: relative;
+  cursor: pointer;
+}
+
+.slides {
+  width: 500%;
+  height: 25vw;
+  display: flex;
+}
+
+.slides input {
+  display: none;
+}
+
+.slide {
+  width: 20%;
+  transition: 2s;
+}
+
+.slide img {
+  width: 100%;
+  height: 25vw;
+}
+
+.navigation-manual {
+  position: absolute;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  bottom: 2rem;
+  z-index: 1;
+}
+
+.manual-btn {
+  border: 2px solid var(--theme-color);
+  padding: 5px;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: 0.5s;
+}
+
+.manual-btn:not(:last-child) {
+  margin-right: 3vw;
+}
+
+.manual-btn:hover {
+  background-color: var(--theme-color);
+}
+
+#radio1:checked ~ .first {
+  margin-left: 0;
+}
+
+#radio2:checked ~ .first {
+  margin-left: -20%;
+}
+
+#radio3:checked ~ .first {
+  margin-left: -40%;
+}
+
+#radio4:checked ~ .first {
+  margin-left: -60%;
+}
+
+.navigation-auto {
+  position: absolute;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  bottom: 2rem;
+  z-index: 1;
+}
+
+.navigation-auto div {
+  border: 2px solid var(--theme-color);
+  padding: 5px;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: 0.5s;
+}
+
+.navigation-auto div:not(:last-child) {
+  margin-right: 3vw;
+}
+
+#radio1:checked ~ .navigation-auto .auto-btn1 {
+  background: var(--theme-color);
+}
+
+#radio2:checked ~ .navigation-auto .auto-btn2 {
+  background: var(--theme-color);
+}
+
+#radio3:checked ~ .navigation-auto .auto-btn3 {
+  background: var(--theme-color);
+}
+
+#radio4:checked ~ .navigation-auto .auto-btn4 {
+  background: var(--theme-color);
+}
+
```

Comment:
all buttons  have same color so try to reduce the repeating code

Reply:
Hii Prathmesh .. these audio button have same colour .. But all will not have the color at same time..
Only one of them is active when corresponding input is clicked ..
So to take that in account with css , i have used repetitive code .. 
Couldn't find any alternative of this except to handle these with javascript

Reply:
oh ok then